src/components/ftbot/PairlistConfigResults.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <b-container fluid>
+    <b-row v-if="whitelist.length > 0">
```

Comment:
I've got quite bad experiences with bootstraps container/row/col model.

it adds margins (and negative margins) at semi-random - which caused quite some problems with offsets in the past.
i've therefore come to apreciate the simple "flex" model more - but if you're sure it works this way without negative side-effects, i'll be more than happy with this model.

Reply:
I haven't really run into any side effects yet at least, and it just feels like a very natural way to lay things out to me.